Makefile

```diff
@@ -1,48 +1,29 @@
-GITHUB_API_TOKEN := ""
-
-##
 # Site
-site-fetch-contributors:
-	GITHUB_API_TOKEN=$(GITHUB_API_TOKEN) node ./site/scripts/fetch-contributors.js
-
-site-build:
-	node ./site/scripts/build-content.js
-
-	mkdir -p ./_site/css
-	./node_modules/.bin/node-sass --include-path ./node_modules ./site/assets/css/main.scss ./_site/css/site.css
 
-	mkdir -p ./_site/js
-	cp -r ./site/assets/js/ ./_site/js/
-	# ./node_modules/.bin/babel ./site/assets/js --out-dir ./_site/js
+site-develop:
+	node ./node_modules/gatsby/dist/bin/gatsby.js develop
 
-	cp -rf ./site/assets/img ./_site/img
 
-site-watch:
```

Comment:
Is it possible to keep `site-watch`? It might be useful for development :)

Reply:
Actually `site-watch` is now merged in the `site-develop` command, so we don't need to run a separate command for this purpose. (I mentioned it in the README)

Reply:
Ok, cool. Sorry, I didn't see it.